### DISCUSSION TITLE
Rev's Adventure Part I

### DISCUSSION THREAD

**Discussion by Revnova** [nzp-team/nzportable#391] [Map Releases]
# [Download Here](https://github.com/user-attachments/files/17333109/Revs-Adventure-1.zip)

This map's custom content are textures and perk machine sounds.

Currently includes:
1. Power
2. 1Mbox Spot
3. 6 Perks
4. 3 Teddys
5. EE Song
6. Coop Spawns
7. 6 Wall Weapons
8. Pack-a-Punch
9. 2 Locations to dive from
10. Grenade/Betty Wall Buys
11. Ending

this map is based on a dream i had a while back. so if there's an impossibilities. just know thats why. i've also added extra lights for anyone wanting to play with realtime shadows and such. the map will spam with FirstPathFind Failure messages. but waypoints are fine. so don't mind it. let me know if there are any issues i can fix. 👍 
Map wasn't made with handhelds in mind. so sorry if it doesn't work on them.
# More Dreams to Come...

Screenshots:
![fte-20231006233439-0](https://github.com/nzp-team/nzportable/assets/34180180/ecef9460-ab2a-4edc-9943-647753d99fdf)
![fte-20231006233511-0](https://github.com/nzp-team/nzportable/assets/34180180/f6f3d27e-8529-49d2-8055-98dabedb680d)
![fte-20231006232912-0](https://github.com/nzp-team/nzportable/assets/34180180/eafe328a-5a73-40e5-b9df-3b6562628275)
![fte-20231006232928-0](https://github.com/nzp-team/nzportable/assets/34180180/c8c6e96e-ba74-44e8-9453-0ea95289e2e7)
![fte-20231006233026-0](https://github.com/nzp-team/nzportable/assets/34180180/c17a4a43-9e22-4200-8773-63138a7b90e6)
![fte-20231006233101-0](https://github.com/nzp-team/nzportable/assets/34180180/ba0b3138-796e-44c6-9721-11b8d862c268)
![fte-20231006233107-0](https://github.com/nzp-team/nzportable/assets/34180180/93cc9fac-e027-4933-8337-4dadceb882b8)

**Comment by MotoLegacy** [comment]
>Map wasn't made with handhelds in mind. so sorry if it doesn't work on them. 

did you test this at all?

**Reply by DerpedCrusader** [reply]
Crashes during loading on Vita

**Reply by irandiaz** [reply]
Brother literally on the PSP and it's a very good map

**Comment by Conmiester** [comment]
I did a quick run of the whole map on PC and it suffers from not only waypoint issues but also the first thing I noticed:

![fte-20231010001034-0](https://github.com/nzp-team/nzportable/assets/63212724/146ca163-6cbb-42bf-8c11-033d2292ea90)
![fte-20231010001133-0](https://github.com/nzp-team/nzportable/assets/63212724/b03b0054-fbcd-483b-9211-3c3a19909f4b)
![fte-20231010001422-0](https://github.com/nzp-team/nzportable/assets/63212724/c1c513f8-4f48-48fb-9027-dd2db18f7625)
Not sure if anyone else had these issues I had but this is what I experienced for the night

**Reply by Revnova** [reply]
I really don't know why waypoints are having issues. I have every waypoint linked. I'll try redoing them later today. I will also move the mbox a bit further from the window to fix buying it through.

**Comment by Revnova** [comment]
# Updated
fix being able to buy the box through the window.
also tried redoing the waypoints too. but it still gives the error.

**Comment by Revnova** [comment]
# Updated Again
-lowered texture resolution and put the textures in its own WAD, so the BSP is smaller now.
-fixed Waypoint causing Errors

**Comment by Revnova** [comment]
what should be a final update.
fixed misaligned textures

**Comment by MotoLegacy** [comment]
Consider editing the post to make the download button more clear/larger, otherwise clear for meeting install standard.

**Reply by pahntomforces-guy** [reply]
how do i my game to play mp3 sfx

**Comment by Revnova** [comment]
# Updated
-Added dive points as per request
-Added PHD Flopper
-Added Grenade/Betty Buys
-Added a zombie spawn in the apartment
-Added a few more props around the map
-Redid Stairway a bit

**Comment by Revnova** [comment]
# Updated
-Added Proper 4 player Spawns (again)
-Gave each player a different starter gun

**Comment by Ellis243454** [comment]
So I love the map but I can't open the power door on 3ds and I don't know why any reason

**Reply by Fiilthy** [reply]
2 years late but I just went to console and used _noclip_.

**Comment by Revnova** [comment]
# Updated
- gave zombies in starting area actual rooms to spawn in from
- changed lighting a bit
- added ending
- replaced the powerline at spawn with two that are more out of the way
- rescaled some models
- moved teddys a bit... (not far from where they originally were)
- changed Sky to look more night time

**Comment by kmorris1077** [comment]
@Revnova I don't see a single teddy on this map.  I have accidentally opened the door to the power source room, am never sure what triggers it.. I see maybe a piece of a teddy in the library room, but shooting it does nothing. (playing on the web)

**Reply by Revnova** [reply]
the 2nd and 3rd teddies are hidden until you hit the 1st one. the 1st one isn't visible either though. did you know that all the barrels on the map can explode? :)

**Reply by kmorris1077** [reply]
This is a fun map, but my feedback is making the teddies hidden AND linked to the door opening is not a great game playing experience.  Maybe there is some lore around this (I'm new to NZP and never played the CoD versions), but if you want to make it a puzzle then do that (maybe the user can see a switch through a window that controls the door that has to be triggered via a barrel detonation cascade that is initiated in another room, spitballing...).  Or if you want a tricky obscure easter egg do that.  But to link a hidden easter egg to core game progression is not great level design IMO. Other than that, love this map. Especially jumping down the stairwell :)

**Comment by leoereno** [comment]
how do i turn power on?

**Reply by leoereno** [reply]
edit: hell yeah
![Screenshot_107](https://github.com/user-attachments/assets/3daa639e-756c-4847-9741-9d55b47e7567)

**Reply by LegendDD44** [reply]
when i explode the barrels nothing happens. what do i do?

**Reply by leoereno** [reply]
throw a grenade close to the bookshelf near jug :)

**Reply by Revnova** [reply]
and make a puzzle completely pointless. you're basically telling me to move them once again.

**Reply by leoereno** [reply]
sorry :/
new locations would be cool tho

**Comment by LegendDD44** [comment]
where are the teddies? i exploded all the barrels.

**Reply by Revnova** [reply]
![image](https://github.com/user-attachments/assets/d9dd44f7-5e37-4c97-b0b4-a7740b21c0c8)
first teddy is found here. only time i'm going to reveal them to you guys. 2nd one you kinda need a scope to find...

**Comment by LegendDD44** [comment]
My best camping spots:
![image](https://github.com/user-attachments/assets/7a19676d-e5bf-4ea9-b949-bdb6676a64a5)
![image](https://github.com/user-attachments/assets/e647698e-0764-4653-b00b-69e3f236aff6)
I like the speed cola one because you can easily get ammo

**Reply by LegendDD44** [reply]
The pictures are glitched. One is behind the mystery box and the other is behind speed cola

**Reply by LegendDD44** [reply]
you can shoot through the speed cola machine

**Reply by mfg471** [reply]
@LegendDD44 although you already mentioned that the pictures are glitched, ah yes, you like the dark.

**Comment by kmorris1077** [comment]
I've found that I don't need to explode the barrels at all on web.  There is what looks like the bottom of a teddy in the library under the top shelf nearest the door. If you explode a grenade near it, the power room opens up.
![Screenshot 2024-10-09 at 1 19 53 PM](https://github.com/user-attachments/assets/8a62bd5a-1dd4-483c-aa78-28953938ae3f)

**Comment by Revnova** [comment]
# Updated
-Made my own model for the lamps outside so they're not blocky
-redid how teddies work
-made it so the final teddy can't be hit by grenades unless open
-made the 1st teddy actually visible
-added a bit of light where the 2nd teddy is so its a bit more visible
-made it a bit brighter outside

**Comment by kmorris1077** [comment]
Much improved! Loved the lighting enhancement in the courtyard. Killed some zombies by diving finally. Found all the teddies.  Looking forward to achieving the ending!

**Reply by kmorris1077** [reply]
Got the ending.  Wouldn't it be nice if a helicopter came down and rescued the player. Just sayin :P

**Comment by 420faded420** [comment]
<img width="1145" height="712" alt="Screenshot 2026-06-21 11 11 45 AM" src="https://github.com/user-attachments/assets/39655c21-450a-4eb3-961e-d755f51e1b41" />

**Reply by M0THRAK** [reply]
Good job!
